docs/standard/native-interop/comwrappers-source-generation.md

```diff
@@ -0,0 +1,196 @@
+---
+title: ComWrappers source generation
+description: Learn about compile-time source generation for ComWrappers in .NET.
+ms.date: 08/25/2023
+---
+
```

Comment:
Add the new pages to the table of contents:
https://github.com/dotnet/docs/blob/70455ed0720e94c61a6452baba6ea50fbe31435a/docs/navigate/advanced-programming/toc.yml#L312